docs/use-signalr-service.md

```diff
@@ -154,6 +156,13 @@ They can be accessed at [`Hub.Context.User`](https://github.com/aspnet/SignalR/b
 - Default value is `5 minutes`
 - This option specifies the longest time in waiting for dynamic scaling service endpoints, that to affect online clients at minimum. Normally the dynamic scale between single app server and a service endpoint can be finished in seconds, while considering if you have multiple app servers and multiple service endpoints with network jitter and would like to ensure client stability, you can configure this value accordingly.
 
+
+#### `MaxPollIntervalInSeconds`
+
+- Default value is `5`
+- This option specifies the longest time in Azure SignalR Service to cleanup idle `LongPolling` connections that useful in scenarios that clients have extra long time between 2 poll requests. Notes that Azure SignalR Service will also clean up connections when cached waiting to write buffer size is greater than `1Mb` to ensure service performance.
```

Comment:
This option defines the max poll interval allowed for `LongPolling` connections in Azure SignalR Service. If the next poll request does not come in within `MaxPollIntervalInSeconds`, Azure SignalR Service cleans up the client connection.